src/ObjCBindings/BindingTypeAttribute.cs

```diff
@@ -0,0 +1,11 @@
+using System;
+using System.Reflection;
+
+namespace ObjCBindings {
+
+	[AttributeUsage (AttributeTargets.Class | System.AttributeTargets.Enum, AllowMultiple = false)]
+	public class BindingTypeAttribute : Attribute {
```

Comment:
Some xml documentation would be nice :)

@dalexsoto also I wonder if we should add the `[Experimental]` attribute to these new attributes types? That way we would be free to make changes to them if we wanted to.

Reply:
Ahh good idea! Let’s do it!

Reply:
+1 on that. Lets add that.